packages/tools/examples/videoSegmentation/index.ts

```diff
@@ -0,0 +1,403 @@
+  `;
+
+content.append(instructions);
+
+const brushInstanceNames = {
+  CircularBrush: 'CircularBrush',
+  CircularEraser: 'CircularEraser',
+  ThresholdBrush: 'ThresholdBrush',
+  DynamicThreshold: 'DynamicThreshold',
+};
+
+const brushStrategies = {
+  [brushInstanceNames.CircularBrush]: 'FILL_INSIDE_CIRCLE',
+  [brushInstanceNames.CircularEraser]: 'ERASE_INSIDE_CIRCLE',
+  [brushInstanceNames.ThresholdBrush]: 'THRESHOLD_INSIDE_CIRCLE',
+  [brushInstanceNames.DynamicThreshold]: 'THRESHOLD_INSIDE_CIRCLE',
+};
+
+const brushValues = [
+  brushInstanceNames.CircularBrush,
+  brushInstanceNames.CircularEraser,
+  brushInstanceNames.ThresholdBrush,
+  brushInstanceNames.DynamicThreshold,
+];
+
+const optionsValues = [
+  ...brushValues,
+  RectangleScissorsTool.toolName,
+  CircleScissorsTool.toolName,
+  PaintFillTool.toolName,
+];
+
+let viewport;
+
+const segmentationIds = ['VIDEO_SEGMENTATION'];
+const segmentationRepresentationUIDs = [];
+const dropDownId = 'SEGMENTATION_DROPDOWN';
+
+function updateSegmentationDropdownOptions(
+  segmentationIds,
+  activeSegmentationId
+) {
+  const dropdown = document.getElementById(
+    'SEGMENTATION_DROPDOWN'
+  ) as HTMLSelectElement;
+
+  dropdown.innerHTML = '';
+
+  segmentationIds.forEach((segmentationId) => {
+    const option = document.createElement('option');
+    option.value = segmentationId;
+    option.innerText = segmentationId;
+    dropdown.appendChild(option);
+  });
+
+  if (activeSegmentationId) {
+    dropdown.value = activeSegmentationId;
+  }
+}
+
+// ============================= //
+addDropdownToToolbar({
+  options: { values: optionsValues, defaultValue: BrushTool.toolName },
+  onSelectedValueChange: (nameAsStringOrNumber) => {
+    const name = String(nameAsStringOrNumber);
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+
+    // Set the currently active tool disabled
+    const toolName = toolGroup.getActivePrimaryMouseButtonTool();
+
+    if (toolName) {
+      toolGroup.setToolDisabled(toolName);
+    }
+
+    if (brushValues.includes(name)) {
+      toolGroup.setToolActive(name, {
+        bindings: [{ mouseButton: MouseBindings.Primary }],
+      });
+    } else {
+      const toolName = name;
+
+      toolGroup.setToolActive(toolName, {
+        bindings: [{ mouseButton: MouseBindings.Primary }],
+      });
+    }
+  },
+});
+
+const thresholdOptions = ['CT Fat: (-150, -70)', 'CT Bone: (200, 1000)'];
+
+addDropdownToToolbar({
+  options: { values: thresholdOptions, defaultValue: thresholdOptions[0] },
+  onSelectedValueChange: (nameAsStringOrNumber) => {
+    const name = String(nameAsStringOrNumber);
+
+    let threshold;
+    if (name === thresholdOptions[0]) {
+      threshold = [-150, -70];
+    } else if (name == thresholdOptions[1]) {
+      threshold = [100, 1000];
+    }
+
+    segmentationUtils.setBrushThresholdForToolGroup(toolGroupId, threshold);
+  },
+});
+
+addButtonToToolbar({
+  title: 'Create New Segmentation on Current Image',
```

Comment:
will this create extra labelmap on the current image? again segmentation vs labelmap